code/modules/slapcrafting/base_steps/slapcraft_step_weapon.dm

```diff
@@ -5,7 +5,7 @@
 	check_types = FALSE
 	check_if_mob_can_drop_item = FALSE
 
-	list_desc = "sharp implement"
+	list_desc = "cutting implement"
```

Comment:
y

Reply:
"cutting implement" sounds more natural then "sharp implement" to me. cutting is what the implement is for. i'll change it if you want but imo it makes sense